common/ripple/userUtils.py

```diff
@@ -332,247 +248,202 @@ async def ban(userID: int) -> None:
-
-    :param userID: userID of the target (restrictee)
-    :return: timestamp
-    """
-
+async def get_freeze_restriction_date(user_id: int) -> int:
+    """Return a user's enqueued restriction date."""
     result = await glob.db.fetch(
         "SELECT frozen FROM users WHERE id = %s",
-        [userID],
+        [user_id],
     )
-
     return result["frozen"] if result else 0
 
 
-async def getFreezeReason(userID: int) -> Optional[str]:
-    result = await glob.db.fetch(
-        "SELECT freeze_reason FROM users WHERE id = %s",
-        [userID],
-    )
-    return result["freeze_reason"] if result["freeze_reason"] else None
-
-
-async def freeze(userID: int, author: int = CHATBOT_USER_ID) -> None:
+async def freeze(user_id: int, *, author_user_id: int = CHATBOT_USER_ID) -> None:
     """
     Enqueue a 'pending' restriction on a user. (7 days)
-    Used for getting liveplays from users already suspected of cheating.
 
-    :param userID: userID of the target (restrictee)
-    :param author: userID of the author (restricter)
+    Used for getting liveplays from users already suspected of cheating.
     """
 
-    await beginFreezeTimer(userID)  # to fix cron bugs
+    await begin_freeze_timer(user_id)  # to fix cron bugs
 
-    author_name = await getUsername(author)
-    target_name = await getUsername(userID)
+    author_name = await get_username_from_id(author_user_id)
+    target_name = await get_username_from_id(user_id)
 
-    await appendNotes(userID, f"{author_name} ({author}) froze this user.")
-    await audit_logs.send_log(author, f"froze {target_name} ({userID}).")
+    await append_cm_notes(user_id, f"{author_name} ({author_user_id}) froze this user.")
+    await audit_logs.send_log(author_user_id, f"froze {target_name} ({user_id}).")
     await audit_logs.send_log_as_discord_webhook(
-        message=f"{author_name} has frozen [{target_name}](https://akatsuki.gg/u/{userID}).",
+        message=f"{author_name} has frozen [{target_name}](https://akatsuki.gg/u/{user_id}).",
         discord_channel="ac_general",
     )
 
 
-async def beginFreezeTimer(userID) -> int:
+async def begin_freeze_timer(user_id: int) -> int:
+    """Enqueue a 'pending' restriction on a user. (in 7 days)"""
     restriction_time = int(time.time() + (86400 * 7))
-
     await glob.db.execute(
         "UPDATE users SET frozen = %s WHERE id = %s",
-        [restriction_time, userID],
+        [restriction_time, user_id],
     )
+    return restriction_time
 
-    return restriction_time  # Return so we can update the time
 
-
-async def unfreeze(userID: int, author: int = CHATBOT_USER_ID, _log=True) -> None:
-    """
-    Dequeue a 'pending' restriction on a user.
-
-    :param userID: userID of the target (restrictee)
-    :param author: userID of the author (restricter)
-    """
+async def unfreeze(
+    user_id: int,
+    *,
+    author_user_id: int = CHATBOT_USER_ID,
+    should_log_to_cm_notes_and_discord: bool = True,

```

Comment:
Also did quite a lot of enforcing of stronger conventions -- e.g. enforcing that keyword args must be used for unclear cases like these.